source/zone.c

```diff
@@ -328,42 +328,6 @@ void* memcpy_vfpu( void* dst, void* src, unsigned int size )
 					:"$8","$9","$10","$11","memory"
 					);
 			}
-			if (size>16)
```

Comment:
This removed section is what resulted in memcpy_vfpy results containing straight zeroes in some cases, like when copying light data for small shell box.